script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -252,6 +254,8 @@ A flash of pain shoots though my skull.
 [sha:f5d83bd282ccb6fd64e4842787833560310f10bf]{
 -- Page 51, Offset 30644.
 -- この痛覚。この危機感は、自分自身に対するものだ。
+// group pr check: sense of impending danger is directed to Shiki himself
```

Comment:
I think Petrikow's comment was that it isn't _directed_, which is what it currently is rendered as, more that it is something that only exists _because it's him_.

Reply:
sugg
> This pain, this sense of impending danger―――they feel rooted within my own identity.